File: Python3/1046 - Tempo de Jogo.py
A, B = map(int, input().split())
if B > A:
    print('O JOGO DUROU {} HORA(S)'.format(B - A))
else:
    C = (24 - A) + B
    print('O JOGO DUROU {} HORA(S)'.format(C))

File: CSharp/1046 - Tempo de Jogo.cs
using System;

class MainClass {
  public static void Main (string[] args) {
    string[] S = Console.ReadLine().Split();
    int A = Convert.ToInt32(S[0]);
    int B = Convert.ToInt32(S[1]);
    int C;
    if (A < B)
      C = B - A;
    else
      C = 24 - A + B;
    Console.WriteLine("O JOGO DUROU " + C + " HORA(S)");
  }
}